src/ScriptEditor/NetscriptDefinitions.d.ts

```diff
@@ -10096,6 +10096,7 @@ interface IStyleSettings {
  */
 interface GameInfo {
   version: string;
```

Comment:
Now that there's two "versions" in here, let's add some docs.
```suggestion
  /** Version as shown in release notes and in the UI. E.g.: "2.8.1dev" */
  version: string;
```